### PR TITLE
(SIMP-4932) Disable release tests by default

### PR DESCRIPTION
**Pull request by jeefberkey**
Previously, some release flavored integration tests would run at every
commit, even when expected to fail. This commit turns them off by
default, but allowing someone to set SIMP_RELEASE_TESTS in the GitLab
repo to run them.

Other changes:
- Condensed test suite READMEs into one

SIMP-4932 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
The tests were skipped, as they should have been: https://gitlab.com/simp/simp-core/pipelines/22514945

File: spec/acceptance/suites/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# simp-core Acceptance and integration tests
+
+In this directory there are tests that can be used to test SIMP in it's
+entirety, including all non ISO components.
+
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/README.md]: which of these tests run 'simp config' and 'simp bootstrap'?  Some do, and some manually affect bootstrapping.

File: spec/acceptance/suites/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+2. Install Ruby 2.1.9 (follow the guides from the link above)
+3. Install bundler: `gem install bundler`
+4. Install other dependencies: `gem install`
+5. Run the tests:
+
+```bash
+bundle exec rake beaker:suites[<suite>,<nodeset>]
+```
+
+There two nodesets per suite, `el7_server` and `el6_server`. They control the
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/README.md]: 3 nodesets.  `default` which is a link to `el7_server` ?

File: spec/acceptance/suites/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+version of EL on the puppetserver.
+
+### default
+
+This test runs `puppet module build` in the root directory of `simp_core`,
+building a development version of the simp/simp_core metamodule. Then, it spins
+up a puppetserver and trys to install the development version of the module,
+using the Puppet Forge as a source for all modules.
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/README.md]: Not PuppetForge.  Installs using R10K from a Puppetfile generated from  Puppetfile.tracking.  (So, git URLs)

File: spec/acceptance/suites/README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+However, if there are unreleased components referenced in the `metadata.json`,
+the `puppet module install` for the module will fail.
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/README.md]: No.  This does not make sense.

Also, this does not run 'simp config' and 'simp bootstrap'.

**Pull request merged**